### DISCUSSION TITLE
ListRecord function deprecated any alternative

### DISCUSSION THREAD

**Discussion by KumarSubham101201** [filamentphp/filament#8899] [Help]
protected function getTableRecordActionUsing(): ?Closure
    {
        return fn (): string => 'view';
    }
    protected function getTableRecordUrlUsing(): ?Closure
    {
        return null;
    }

in v3 these two function is deprecated any aternative

**Comment by leandrocfe** [comment]
```php
public static function table(Table $table): Table
{
    return $table
    ->recordUrl('xx')
    ->recordAction('xx')
    ...
```
?
